packages/web/docs/src/content/gateway/deployment/node-frameworks/nestjs.mdx

```diff
@@ -0,0 +1,62 @@
+---
+description:
+  Nest (Nest JS) is a progressive Node.js framework for building efficient, reliable and scalable
+  server-side applications.
+---
+
+import { Callout } from '@theguild/components'
+
+# Integration with NestJS
+
+[Nest (Nest JS)](https://nestjs.com) is a progressive Node.js framework for building efficient,
+reliable and scalable server-side applications.
+
+Hive Gateway provides its own Nest GraphQL Driver that support building
+[GraphQL Federation](https://the-guild.dev/graphql/hive/federation) gateways including
+[subscription support](/docs/gateway/subscriptions) that do not exist in the default Nest Federation
+drivers.
```

Comment:
Some grammar fixes and removed the last part of the sentence, there's no need to state that subscriptions dont exist in Nest's Federation driver.

```suggestion
Hive Gateway provides its own Nest GraphQL Driver that supports building
[GraphQL Federation](https://the-guild.dev/graphql/hive/federation) gateways including
[subscriptions support](/docs/gateway/subscriptions).
```

Reply:
Isn't it a good selling point for Hive Gateway driver compared to default options?

Reply:
I think it's enough to say that we support subscriptions, no need to connect ourselves to Nest. Also, this might change in the future and not saying anything about Nest keeps the docs up-to-date on its own.

Reply:
In Nest docs, it says gateway doesn't support subscriptions which might be misleading on our end like we also do not support. My point is to emphasize that we support unlike the default existing options for the Federation Gateway with Nest.